mycroft/client/speech/main.py

```diff
@@ -80,8 +81,7 @@ def handle_speak(event):
 def handle_complete_intent_failure(event):
     LOG.info("Failed to find intent.")
     # TODO: Localize
```

Comment:
You can remove this TODO comment with this change

Reply:
Fixed!